p2p/ipld/plugin/Makefile

```diff
@@ -1,34 +0,0 @@
-# Force Go Modules
-GO111MODULE = on
-
-GOCC ?= go
-GOFLAGS ?=
-
-# If set, override the install location for plugins
-IPFS_PATH ?= $(HOME)/.ipfs
-
-# If set, override the IPFS version to build against. This _modifies_ the local
-# go.mod/go.sum files and permanently sets this version.
-IPFS_VERSION ?= $(lastword $(shell $(GOCC) list -m github.com/ipfs/go-ipfs))
-
-# make reproducible
-ifneq ($(findstring /,$(IPFS_VERSION)),)
-# Locally built go-ipfs
-GOFLAGS += -asmflags=all=-trimpath="$(GOPATH)" -gcflags=all=-trimpath="$(GOPATH)"
-else
-# Remote version of go-ipfs (e.g. via `go get -trimpath` or official distribution)
-GOFLAGS += -trimpath
-endif
-
-.PHONY: install build
-
-lazyledger-plugin.so: plugin.go go.mod
-	$(GOCC) build $(GOFLAGS) -buildmode=plugin -o "$@" "$<"
-	chmod +x "$@"
-
-build: lazyledger-plugin.so
-	@echo "Built against" $(IPFS_VERSION)
-
-install: build
-	mkdir -p "$(IPFS_PATH)/plugins/"
-	cp -f lazyledger-plugin.so "$(IPFS_PATH)/plugins/lazyledger-plugin.so"
```

Comment:
I don't see where this logic moved to, do we not need it anymore?

Reply:
I removed it as considered stale, but can be wrong

Reply:
I think we still need to be able to easily install the plugin. If we fixed the naming conflicts of some of the commands/vars, we should be able to import it into the root `Makefile`

Reply:
It was there to install and test the plugin with an IPFS node (without running a tendermint node). It was a bit tricky to get it to work properly though (as IPFS will complain about a different versions of go-packages). Thinking about this now, that was another reason why this was a separate module instead of just a package 🤔 As it allowed using replace in go.mod to enforce the same versions of go modules in the plugin as in the particular ipfs version you'd want to use the plugin with. Although this isn't really the main use-case of the plugin, it was still use-ful to test it on the commandline.
Does the Makefile still work though (maybe with some changes)?